subjects/graphql/index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Graphql Doc</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<style>
+
+:root {
+  --color: hsl(0, 0%, 65%);
+}
```

Comment:
what is all this ?
Did you recode your own [GraphiQL](https://github.com/graphql/graphiql) ?